staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/testdata/invalid-configs/aes/invalid-key.yaml

```diff
@@ -1,13 +0,0 @@
-kind: EncryptionConfiguration
```

Comment:
were all of these tests moved somewhere else, or were they removed?

Reply:
A number of tests (and their corresponding test data files) have been removed from config_test.go. Such tests dealt with the logic of validating EncryptionConfig.

Reply:
were corresponding validation tests added?

Reply:
Yes.